deploy/operator/ztp/deploy_sno.sh

```diff
@@ -0,0 +1,89 @@
+export DS_OPENSHIFT_VERSION="${DS_OPENSHIFT_VERSION:-openshift-v4.8.0}"
+export OPENSHIFT_INSTALL_RELEASE_IMAGE="${OPENSHIFT_INSTALL_RELEASE_IMAGE:-quay.io/openshift-release-dev/ocp-release:4.8.0-fc.3-x86_64}"
+export ASSISTED_CLUSTER_DEPLOYMENT_NAME="${ASSISTED_CLUSTER_DEPLOYMENT_NAME:-assisted-test-cluster}"
+export ASSISTED_AGENT_CLUSTER_INSTALL_NAME="${ASSISTED_AGENT_CLUSTER_INSTALL_NAME:-assisted-agent-cluster-install}"
+export ASSISTED_INFRAENV_NAME="${ASSISTED_INFRAENV_NAME:-assisted-infra-env}"
+export ASSISTED_PULLSECRET_NAME="${ASSISTED_PULLSECRET_NAME:-assisted-pull-secret}"
+export ASSISTED_PULLSECRET_JSON="${ASSISTED_PULLSECRET_JSON:-/home/test/dev-scripts/pull_secret.json}"
+export ASSISTED_PRIVATEKEY_NAME="${ASSISTED_PRIVATEKEY_NAME:-assisted-ssh-private-key}"
+export EXTRA_BAREMETALHOSTS_FILE="${EXTRA_BAREMETALHOSTS_FILE:-/home/test/dev-scripts/ocp/ostest/extra_baremetalhosts.json}"
+# TODO: make SSH public key configurable
+
+echo "Step configuration:"
+env | sort
+
+echo "Enabling configuration of BMH resources outside of openshift-machine-api namespace"
+oc patch provisioning provisioning-configuration --type merge -p '{"spec":{"watchAllNamespaces": true}}'
+
+echo "Extract information about extra worker nodes..."
+jq --raw-output '.[] | .name + " " + .ports[0].address + " " + .driver_info.username + " " + .driver_info.password + " " + .driver_info.address' "${EXTRA_BAREMETALHOSTS_FILE}" \
+    | while read BMH_NAME MAC_ADDRESS username password ADDRESS ; do
+
+    ENCODED_USERNAME=$(echo -n "${username}" | base64)
+    ENCODED_PASSWORD=$(echo -n "${password}" | base64)
+
+echo "Running Ansible playbook to create kubernetes objects"
+export BMH_NAME MAC_ADDRESS ENCODED_USERNAME ENCODED_PASSWORD ADDRESS
+ansible-playbook assisted-installer-crds-playbook.yaml
+
+echo "Metal3 pods"
+oc get pods -A | grep metal3
+
+echo "VM status"
+virsh list --all
+vm=""
+vm=$(virsh list --all | grep "shut off" | awk '{print $2}')
+echo "Offline VM for ZTP is $vm"
+
+echo "Creating Assisted Installer Pull Secret"
+oc create secret generic ${ASSISTED_PULLSECRET_NAME} --from-file=.dockerconfigjson=${ASSISTED_PULLSECRET_JSON} --type=kubernetes.io/dockerconfigjson -n ${ASSISTED_NAMESPACE}
+
+echo "Creating Assisted Installer Private Key"
+oc create secret generic ${ASSISTED_PRIVATEKEY_NAME} --from-file=ssh-privatekey=/root/.ssh/id_rsa --type=kubernetes.io/ssh-auth -n ${ASSISTED_NAMESPACE}
+
+echo "Creating Assisted Installer Cluster Image Set"
+oc create -f generated/clusterImageSet.yaml
+
+echo "Creating Assisted Installer SNO ClusterDeployment"
+oc create -f generated/clusterDeployment.yaml
+
+echo "Creating Assisted Installer InfraEnv"
+oc create -f generated/infraEnv.yaml
+
+echo "Creating Assisted Installer AgentClusterInstall"
+oc create -f generated/agentClusterInstall.yaml
+
+echo "Creating new baremetal host using baremetal operator"
+oc create -f generate/baremetalHost.yaml
```

Comment:
I don't think the `echo`s are needed. The `oc create` is verbose and outputs what it created